functions/vercel/api/get/[table]/[id].ts

```diff
@@ -0,0 +1,48 @@
+/**
+ *
+ * @method GET
+ * @query ID
+ * @return JSON
+ *
+ * @endpoint /get/[id]
+ */
+import 'isomorphic-fetch'
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+import type { TableNames } from '../../../_lib/types'
+import { getXataClient } from '../../../_lib/xata.codegen'
+// import { XataRecord } from '@xata.io/client'
+
+export default async function handler(
+  request: VercelRequest,
+  response: VercelResponse
+) {
+  const table = request.query.table as TableNames
+  const id = request.query.id as string
+  // const id = 'rec_cci5p8miqtgok3idj8b0'
```

Comment:
🤔

Reply:
just a hint in case anyone wants to try the request to our deployed example 🥚🐇
Should I remove it?